middleware.ts

```diff
@@ -90,7 +90,10 @@ export async function middleware(req: NextRequest) {
 
     if (list && list.data) {
       return NextResponse.redirect(
-        new URL(`/workspaces/${list.data.workspaces?.id}/contributor-insights/${listId}/${rest.join("/")}`, req.url)
+        new URL(
+          `/workspaces/${list.data.workspaces?.workspace_id}/contributor-insights/${listId}/${rest.join("/")}`,
```

Comment:
Was this not working prior to this fix because the type was wrong, so the incorrect property was being used?

Reply:
I thought I tested after being deployed prior to the fix, but after inspecting the data I noticed the incorrect property being used.